Eatery/Controllers/Eateries/EateriesMenuViewController.swift

```diff
@@ -136,12 +137,18 @@ class EateriesMenuViewController: ImageParallaxScrollViewController {
             self?.view.layoutIfNeeded()
         }
     }
-
     func createHeroModifiers(_ groups: HeroModifierGroups...) -> [HeroModifier] {
         [.useGlobalCoordinateSpace, .whenPresenting(.delay(0.15))]
             + (groups.contains(.fade) ? [.fade] : [])
             + (groups.contains(.translate) ? [.translate(y: 32), .timingFunction(.deceleration)] : [])
     }
+    // 0.15 is default delay used across all presenting hero animations
+    // Custom delay is useful when adjusting animation timings
+    func createHeroModifiersWithDelay(delay: Double = 0.15, _ groups: HeroModifierGroups...) -> [HeroModifier] {
+        [.useGlobalCoordinateSpace, .whenPresenting(.delay(delay))]
+            + (groups.contains(.fade) ? [.fade] : [])
+            + (groups.contains(.translate) ? [.translate(y: 32), .timingFunction(.deceleration)] : [])
```

Comment:
What is 32? Magic numbers sus

Reply:
This also appears on 143, might wanna factor that one out too